src/lib/components/wizard-navigation-bar.component.vertical.less

```diff
@@ -1,15 +1,15 @@
 /*
- color definitions
- */
+color definitions
+*/
 @wz-color-default: #E6E6E6;
 @wz-color-current: #808080;
 @wz-color-done: #339933;
 @wz-color-optional: #38ef38;
 @wz-color-editing: #FF0000;
 
 /*
- dot definitions
- */
+dot definitions
```

Comment:
I know this isn't really a part of this PR, but can you please change the style of the old comments:
```css
/*
...
*/
```
to fit with the new ones:
```css
// ...
```

Reply:
f982afc